### DISCUSSION TITLE
Account primary email deleted so can't verify new emails added

### DISCUSSION THREAD

**Discussion by mpt25** [community/community#29709] [Other Feature Feedback, Questions, & Ideas]
Hello, I made my github account last summer when I was working in an internship, and started using the account as my personal one. But today I tried adding my personal and school emails to the profile, but it asked for a verification from my primary email (email from my old internship). The email has been deleted by that organization as I no loner work there or use it, so I am unable to verify any new emails I add. I am looking to add a new email without having to verify it from the deleted account.

**Comment by airtower-luna** [comment]
That's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help with account details. Good luck!
